coderd/database/migrations/000226_provisioner_keys.up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE provisioner_keys (
+    id uuid PRIMARY KEY,
+    created_at timestamptz NOT NULL,
+    organization_id uuid NOT NULL REFERENCES organizations (id) ON DELETE CASCADE,
+    name varchar(64) NOT NULL,
+    hashed_secret bytea NOT NULL,
+    UNIQUE (organization_id, name)
```

Comment:
We should make this unique on the `lower(name)` like workspaces and templates.

An example: https://github.com/coder/coder/blob/4d7ee0b5a18a122c1a565b04644f3ea25b40464a/coderd/database/migrations/000003_workspaces.up.sql#L16-L16